CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ Prefix the change with one of these keywords:
 - _Fixed_: for any bug fixes.
 - _Security_: in case of vulnerabilities.
 
+## [new version]
+- Changed: Hooks should be exported directly [#1120]
```

Comment:
I would suggest something like: "Breaking: `hooks` object is not exported anymore. You can import hooks now directly, for example `import { useFocus } from '@amsterdam/asc-ui' `"

Reply:
gedaan